src/components/_tests_/DateInput.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { shallow } from "enzyme";
+import { createElement } from "react";
+import * as FaCalendar from "react-icons/lib/fa/calendar";
+
+import { DateInput, DateInputProps } from "../DateInput";
+
+describe("DatePicker", () => {
+    const shallowRenderDatePicker = (props: DateInputProps) => shallow(createElement(DateInput, props));
+    const defaultProps: DateInputProps = {
+        dateAttribute: "date",
+        onChange: jasmine.createSpy("change"),
+        onClick: jasmine.createSpy("click"),
```

Comment:
change `onClick: jasmine.createSpy("click"),`  to `onClick: jasmine.any(Function)`